From 3ca104e1b938f5f4bef818f6e22c7fe0427eab9b Mon Sep 17 00:00:00 2001
From: wdc <wdc@rock-chips.com>
Date: Fri, 18 Apr 2014 12:55:29 +0800
Subject: [PATCH] Revert "rk3288:keep camera pwen high alwaysly to avoid
 leakage of lcdc electricity"

This reverts commit 0c0794d1d6f52af604cf83fb32b8351c5dc59ea2.
---
 drivers/media/video/rk_camsys/camsys_internal.h | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/drivers/media/video/rk_camsys/camsys_internal.h b/drivers/media/video/rk_camsys/camsys_internal.h
index 2e5a577cf181..e652aec383e6 100755
--- a/drivers/media/video/rk_camsys/camsys_internal.h
+++ b/drivers/media/video/rk_camsys/camsys_internal.h
@@ -290,13 +290,9 @@ static inline int camsys_sysctl_extdev(camsys_extdev_t *extdev, camsys_sysctrl_t
                 gpio_set_value(gpio->io, gpio->active);
                 camsys_trace(1,"Sysctl %d success, gpio(%d) set %d",devctl->ops, gpio->io, gpio->active);
             } else {
-            	if(CamSys_PwrEn != devctl->ops){
-	                gpio_direction_output(gpio->io, !gpio->active);
-	                gpio_set_value(gpio->io, !gpio->active);
-	                camsys_trace(1,"Sysctl %d success, gpio(%d) set %d",devctl->ops, gpio->io, !gpio->active);
-            		}else{
-            		camsys_trace(1,"don't poweroff CamSys_PwrEn now \n");
-        		}
+                gpio_direction_output(gpio->io, !gpio->active);
+                gpio_set_value(gpio->io, !gpio->active);
+                camsys_trace(1,"Sysctl %d success, gpio(%d) set %d",devctl->ops, gpio->io, !gpio->active);
             }
         } else {
             camsys_err("Sysctl %d failed, because gpio is NULL!",devctl->ops);
-- 
2.35.3

